lisa/operating_system.py

```diff
@@ -58,6 +68,15 @@ class OsInformation:
     full_version: str = "Unknown"
 
 
+class PackageInformation:
+    name: str
+    release: str
+
+    def __init__(self, name: str, release: str) -> None:
+        self.name = name
+        self.release = release
```

Comment:
Can it use the VersionInfo class like kernel version? So it can be comparable easily.